By default, prefer XVideo output module to OpenGL, as OpenGL module requires much more resources when using software emulation

--- vlc-3.0.7.1/modules/video_output/opengl/display.c.1	2019-07-24 11:44:59.489320872 +0000
+++ vlc-3.0.7.1/modules/video_output/opengl/display.c	2019-07-24 11:46:03.253101501 +0000
@@ -48,7 +48,7 @@
 # define MODULE_VARNAME "gles2"
     set_shortname (N_("OpenGL ES2"))
     set_description (N_("OpenGL for Embedded Systems 2 video output"))
-    set_capability ("vout display", 265)
+    set_capability ("vout display", 165)
     set_callbacks (Open, Close)
     add_shortcut ("opengles2", "gles2")
     add_module ("gles2", "opengl es2", NULL,
@@ -62,7 +62,7 @@
     set_description (N_("OpenGL video output"))
     set_category (CAT_VIDEO)
     set_subcategory (SUBCAT_VIDEO_VOUT)
-    set_capability ("vout display", 270)
+    set_capability ("vout display", 170)
     set_callbacks (Open, Close)
     add_shortcut ("opengl", "gl")
     add_module ("gl", "opengl", NULL,
